packages/samples/headless-ssr-commerce/next.config.mjs

```diff
@@ -9,6 +9,9 @@ const nextConfig = {
       },
     ];
   },
+  images: {
+    domains: ['images.barca.group'],
+  },
```

Comment:
This is just necessary to use nextjs `<Image/>`

Reply:
Ideally you should use images.remotePatterns instead, images.domains is deprecated since nextjs 14 https://nextjs.org/docs/app/api-reference/components/image#domains